com.unity.render-pipelines.high-definition/Runtime/RenderPipeline/HDRenderPipelineAsset.Migration.cs

```diff
@@ -136,6 +137,12 @@ enum Version
                 FrameSettings.MigrateVirtualTexturing(ref data.m_RenderingPathDefaultCameraFrameSettings);
                 FrameSettings.MigrateVirtualTexturing(ref data.m_RenderingPathDefaultBakedOrCustomReflectionFrameSettings);
                 FrameSettings.MigrateVirtualTexturing(ref data.m_RenderingPathDefaultRealtimeReflectionFrameSettings);
+            }),
+            MigrationStep.New(Version.UpdateMSAA, (HDRenderPipelineAsset data) =>
+            {
+                FrameSettings.MigrateMSAA(ref data.m_RenderingPathDefaultCameraFrameSettings);
+                FrameSettings.MigrateMSAA(ref data.m_RenderingPathDefaultBakedOrCustomReflectionFrameSettings);
+                FrameSettings.MigrateMSAA(ref data.m_RenderingPathDefaultRealtimeReflectionFrameSettings);
```

Comment:
So if I get it well, if it was true, now it get FromHDRPAsset. But this is the HDRPAsset. I'm confused.
Should it be MSAA2x here?